examples/react_webpack_hotreload/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "v5",
```

Comment:
it's a strange name
also `"private": true` is probably wise

Reply:
my project code, will replace